Documentation/building/windows-instructions.md

```diff
@@ -14,6 +14,8 @@ The following free downloads are compatible:
 
 Note: In order to build our C++ projects be sure to select "Programming Languages | Visual C++ | Common Tools for Visual C++ 2015" while installing VS 2015 (or modify your install to include it).
 
+[CMake](https://cmake.org/) is required to build the native libraries for Windows. To build these libraries cmake must be installed from [the CMake download page](https://cmake.org/download/) and added to your path.
```

Comment:
Not yet. We use 2.8.1.2 on Ubuntu (which uses all of the same stuff as on Windows) and 3.1 in fxkit. The latest (3.4.1) also works without issue. The version so far doesn't matter.

It's possible future updates to CMake might break something that we rely on though. If that's a worry then I can make the docs instruct to download 3.4.1, but it seems a bit... arbitrary.